README.md

```diff
@@ -1,6 +1,168 @@
 [![License](https://img.shields.io/badge/license-Apache--2.0-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0)
 [![Twitter Follow](https://img.shields.io/twitter/follow/strimziio.svg?style=social&label=Follow&style=for-the-badge)](https://twitter.com/strimziio)
 
-# Strimzi Template repository
+# Strimzi Drain Cleaner
 
-This repository is currently empty
+Strimzi Drain Cleaner is utility which helps with moving the Kafka pods deployed by [Strimzi](https://strimzi.io/) from nodes which are being drained.
+It is useful if you want the Strimzi operator to move the pods instead of Kubernetes itself.
+The advantage of this approach is that the Strimzi operator makes sure that no pods become under-replicated during the node draining.
+To use it:
+
+* Deploy Kafka using Strimzi and configure the `PodDisruptionBudgets` for Kafka and Zookeeper to have `maxUnavailable` set to `0`.
+This will block Kubernetes from moving the pods on their own.
+  
+```yaml
+apiVersion: kafka.strimzi.io/v1beta2
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    replicas: 3
+    listeners:
+      - name: plain
+        port: 9092
+        type: internal
+        tls: false
+      - name: tls
+        port: 9093
+        type: internal
+        tls: true
+    config:
+      offsets.topic.replication.factor: 3
+      transaction.state.log.replication.factor: 3
+      transaction.state.log.min.isr: 2
+    storage:
+      type: jbod
+      volumes:
+      - id: 0
+        type: persistent-claim
+        size: 100Gi
+        deleteClaim: false
+    template:
+      podDisruptionBudget:
+        maxUnavailable: 0
+  zookeeper:
+    replicas: 3
+    storage:
+      type: persistent-claim
+      size: 100Gi
+      deleteClaim: false
+    template:
+      podDisruptionBudget:
+        maxUnavailable: 0
+  entityOperator:
+    topicOperator: {}
+    userOperator: {}
+```
+
+* Deploy the Strimzi Drain Cleaner
+* Drain the node with some Kafka or Zookeeper pods using the `kubectl drain` command
```

Comment:
```suggestion
* Drain the node with some Kafka or ZooKeeper pods using the `kubectl drain` command
```

Reply:
Maybe also adding a link to the Kubernetes draining command, i.e. https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/